package.json

```diff
@@ -36,7 +36,7 @@
     "babel-plugin-styled-components": "^1.11.1",
     "classnames": "^2.2.6",
     "crypto": "^1.0.1",
-    "gatsby": "^2.24.67",
+    "gatsby": ">=1",
```

Comment:
What does this do?
I think it would be better to lock it on a certain version as there could be breaking changes